lib/vmfloaty/abs.rb

```diff
@@ -411,4 +411,15 @@ def self.valid_json?(json)
   rescue TypeError, JSON::ParserError => e
     return false
   end
+
+  # when missing, adds the required api/v2 in the url
+  def self.supported_abs_url(url)
+    expected_ending = "api/v2"
+    if !url.include?(expected_ending)
```

Comment:
Shouldn't it be end_with?

```suggestion
    if !url.end_with?(expected_ending)
```

Reply:
no because it might end with "api/v2/" and in that case I do not touch it
```
irb(main):010:0> supported_abs_url("http://foo")
=> "http://foo/api/v2"

irb(main):011:0> supported_abs_url("http://foo/")
=> "http://foo/api/v2"

irb(main):012:0> supported_abs_url("http://foo/api/v2/")
=> "http://foo/api/v2/"

irb(main):013:0> supported_abs_url("http://foo/api/v2")
=> "http://foo/api/v2"
```